src/collection.ts

```diff
@@ -780,13 +780,14 @@ export default class Collection {
     }
     const { data: changes } = await this.bucket.listHistory<T>({
       pages: Infinity, // all pages up to target timestamp are required
-      sort: "-target.data.last_modified",
+      sort: "target.data.last_modified",
       filters: {
         resource_name: "record",
         collection_id: this.name,
         "max_target.data.last_modified": String(at), // eq. to <=
       },
     });
+    console.log(`${changes.length} history entries found.`);
```

Comment:
We should probably figure out a system to allow for optional debugging statements, but for now let's remove calls to `console.log` and `console.debug`.